CHANGELOG.md

```diff
@@ -4,6 +4,63 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
+* Fix ICE due to mishandling of Aggregate rvalue for raw pointers to trait objects by @carolynzech in https://github.com/model-checking/kani/pull/3636
+* Fix loop contracts transformation when loops in branching by @qinheping in https://github.com/model-checking/kani/pull/3640
+* Move any_slice_from_array to kani_core by @qinheping in https://github.com/model-checking/kani/pull/3646
+* Implement `Arbitrary` for `Range*` by @c410-f3r in https://github.com/model-checking/kani/pull/3666
+* codegen: Ask the layout if it is uninhabited, not its impl detail by @workingjubilee in https://github.com/model-checking/kani/pull/3675
+* Update dependencies following Audit workflow failure. by @remi-delmas-3000 in https://github.com/model-checking/kani/pull/3680
+* Harness output individual files by @Alexander-Aghili in https://github.com/model-checking/kani/pull/3360
+* Add support for float_to_int_unchecked by @zhassan-aws in https://github.com/model-checking/kani/pull/3660
+* Change `same_allocation` to accept wide pointers by @celinval in https://github.com/model-checking/kani/pull/3684
+* Derive `Arbitrary` for enums with a single variant by @AlgebraicWolf in https://github.com/model-checking/kani/pull/3692
+* Apply loop contracts only if there exists some usage by @qinheping in https://github.com/model-checking/kani/pull/3694
+* Update verify-std-check workflow to enable loop contracts by @qinheping in https://github.com/model-checking/kani/pull/3705
+* Add support for f16 and f128 in float_to_int_unchecked intrinsic by @zhassan-aws in https://github.com/model-checking/kani/pull/3701
+* Revert "Ignore derivative in Cargo deny" by @qinheping in https://github.com/model-checking/kani/pull/3712
+* Fix codegen for rvalue aggregate raw pointer to an adt with slice tail by @carolynzech in https://github.com/model-checking/kani/pull/3644
+* Improve Kani handling of function markers by @celinval in https://github.com/model-checking/kani/pull/3718
+* Enable contracts for const generic functions by @qinheping in https://github.com/model-checking/kani/pull/3726
+* List Subcommand Improvements by @carolynzech in https://github.com/model-checking/kani/pull/3729
+* add support for enum, struct, tuple in llbc backend by @thanhnguyen-aws in https://github.com/model-checking/kani/pull/3721
+* Fix issues with how we compute DST size by @celinval in https://github.com/model-checking/kani/pull/3687
+* Fix size and alignment computation for intrinsics by @celinval in https://github.com/model-checking/kani/pull/3734
+* Cleanup a few internal compiler deps by @celinval in https://github.com/model-checking/kani/pull/3739
+* Add a Kani function that checks if the range of a float is valid for conversion to int by @zhassan-aws in https://github.com/model-checking/kani/pull/3742
+* Setup/CI: cleanup Ubuntu 18.04 and cbmc-viewer left-overs and enable 24.04 by @tautschnig in https://github.com/model-checking/kani/pull/3758
```

Comment:
Maybe move this to "Major Changes" and just rename it to "Add support for Ubuntu 24.04"?